practice_05.25.19_interface/Task2_BankAccount/Bank.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace practice_05._25._19_interface.Task2_BankAccount
+{
+    public class Bank
+    {
+        private readonly List<BankAccount> accounts;
+
+        public Bank()
+        {
+            accounts = new List<BankAccount>();
+        }
+
+        public List<BankAccount> ManageAccount()
```

Comment:
Where is logic for managing accounts: put money, withdraw, etc?

Reply:
fixed, but need check again